polkadot/node/network/statement-distribution/src/v2/cluster.rs

```diff
@@ -424,6 +425,25 @@ impl ClusterTracker {
 	fn is_in_group(&self, validator: ValidatorIndex) -> bool {
 		self.validators.contains(&validator)
 	}
+
+	/// Dumps pending statement for this cluster.
+	///
+	/// Normally we should not have any pending statements for our cluster,
+	/// but if we do for long periods of time something bad happened which
+	/// needs to be investigated.
+	pub fn dump_pending_statements(&self, parent_hash: Hash) {
```

Comment:
can we rename it `warn_if_too_many_pending_statements` or something like that?
otherwise it sounds like an unconditional print

Reply:
Done.